themes/base.theme.bash

```diff
@@ -16,8 +16,12 @@ SCM_THEME_BRANCH_GONE_PREFIX=' ⇢ '
 SCM_THEME_CURRENT_USER_PREFFIX=' ☺︎ '
 SCM_THEME_CURRENT_USER_SUFFIX=''
 
-CLOCK_CHAR='☆'
-THEME_CLOCK_CHECK=${THEME_CLOCK_CHECK:=true}
+CLOCK_CHAR='⌚'
+THEME_CLOCK_CHAR_COLOR=${THEME_CLOCK_CHAR_COLOR:="${red}"}
+THEME_SHOW_CLOCK=${THEME_CLOCK_CHECK:-true}
```

Comment:
Yeah, I missed that one. Will fix it.

By the way, if you are up for standardizing the theme variables like these, I can make changes to other themes and work on the wiki page for it.